blocks/v2-stories-carousel/v2-stories-carousel.css

```diff
@@ -0,0 +1,258 @@
+:root {
+  --stories-carousel-img-height: 342px;
+  --stories-carousel-img-width: 85%;
+  --stories-carousel-gap: 12px;
+}
+
+/* Full width block */
+main .section.v2-stories-container {
```

Comment:
I don't see `v2-stories-container` in the code or example page. Is it still needed? If so we shouldn't do it in the section. A section can contain more than one component and adding a class that changes styles for the whole section has an impact on the other block in this section. If you need to set the padding to 0 you should add a section metadata class so it will be clear that it is intentional to change the padding for the section.
With the `v2-stories-container` class it is not possible to understand the behavior from the content author's perspective, but a class in the section metadata like: 'padding-0' is clear.

About section metadata:
https://www.hlx.live/developer/block-collection/section-metadata